cmd/kpod/ps.go

```diff
@@ -177,38 +180,170 @@ func psCmd(c *cli.Context) error {
 		namespace: c.Bool("namespace"),
 	}
 
-	// all, latest, and last are mutually exclusive. Only one flag can be used at a time
-	exclusiveOpts := 0
-	if opts.last >= 0 {
-		exclusiveOpts++
-	}
-	if opts.latest {
-		exclusiveOpts++
-	}
-	if opts.all {
-		exclusiveOpts++
-	}
-	if exclusiveOpts > 1 {
-		return errors.Errorf("Last, latest and all are mutually exclusive")
+	var filterFuncs []libpod.ContainerFilter
+	// When we are dealing with latest or last=n, we need to
+	// get all containers.
+	if !opts.all && !opts.latest && opts.last < 1 {
+		// only get running containers
+		filterFuncs = append(filterFuncs, func(c *libpod.Container) bool {
+			state, _ := c.State()
+			if state.String() == libpod.ContainerStateRunning.String() {
```

Comment:
I believe you can just say

return  state.String() == libpod.ContainerStateRunning.String()

Reply:
done